src/DiffEqOperators.jl

```diff
@@ -56,5 +60,6 @@ export DirichletBC, Dirichlet0BC, NeumannBC, Neumann0BC, RobinBC, GeneralBC, Mul
        MultiDimDirectionalBC, ComposedMultiDimBC,
        compose, decompose, perpsize
 export GhostDerivativeOperator
+export DiagonalOperator, IdentityOperator
```

Comment:
These don't make a lot of sense. Isn't `DiagonalOperator` just an array operator with a `Diagonal` array? And `IdentitiyOperator`: does it do more than `I`?

Reply:
My apologies, I didn't mean to make this part of the PR. Rewinding this branch (and getting rid of the DiagonalOperator and IdentityOperator).